package.json

```diff
@@ -40,7 +40,7 @@
     "eslint": "3.18.0",
     "eslint-config-xo": "0.18.1",
     "eslint-plugin-security": "^1.3.0",
-    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp": "github:gulpjs/gulp#3f5aba2",
```

Comment:
> Which means we don't get new free stuff (features, bugfixes).

Jup. It’s a tradeoff. But since our tasks are running for now, I prefer to keep ’em running instead of getting bugs or breaking changes from a dev branch 😊 

Hope the release a alpha/beta on npm soon. See [v4 Roadmap](https://github.com/gulpjs/gulp/issues/1945)